src/js/components/stock-movement/outbound/StockMovementOutboundTable.jsx

```diff
@@ -43,6 +43,8 @@ const StockMovementOutboundTable = ({
     deleteConfirmAlert,
   } = useOutboundListTableData(filterParams);
 
+  useTranslation('requisitionStatus');
```

Comment:
nitpicky: I think this call should be placed in the `useOutboundListTableData` hook, but no need to change.